bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Flight/Profile.cpp

```diff
@@ -56,6 +56,8 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Flight_Profile(pybind11::module& aMo
         .value("GeocentricNadir", Profile::TargetType::GeocentricNadir, "Geocentric nadir")
         .value("GeodeticNadir", Profile::TargetType::GeodeticNadir, "Geodetic nadir")
         .value("Trajectory", Profile::TargetType::Trajectory, "Trajectory")
```

Comment:
can you add a depracation warning here?

Reply:
Not sure how to add it at the enum level, added a comment